src/main/java/core/basesyntax/AttributesDrawing.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AttributesDrawing {
+    void draw();
+}
```

Comment:
The name `AttributesDrawing` suggests that it's drawing attributes, which might not be the intent. According to the checklist, interface names should not be verbs. A better name might be `Drawable` or `FigureDrawer`, which clearly indicates that this interface is for objects that can be drawn.